### PR TITLE
Solving #1738

### PR DESCRIPTION
**Pull request by MKergall**

### CODE REVIEW

File: osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java [hunk 1]
```diff
@@ -571,7 +571,8 @@ protected void drawAt(final Canvas pCanvas, final int pX, final int pY, final fl
             pCanvas.save();
             pCanvas.rotate(pOrientation, pX, pY);
         }
-        if (mIcon instanceof BitmapDrawable) { // optimization 3: (about 15% faster)
+        /*
```
Comment (monsieurtanuki) [inline osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java:574]: I would add a single comment like "Unfortunate optimization with displayed size side effects"

**Review by monsieurtanuki — APPROVED** [review]
Thank you @MKergall!
I just have one minor comment: please read it, feel free to ignore it, and then merge this PR.

File: osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java (hunk 1 above)
Comment (MKergall) [reply osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java:574]: Sure. But I don't know how to add that change in the PR :-(

Comment (MKergall) [reply osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java:574]: Found!

**Pull request merged**